servers/physics_3d/soft_body_3d_sw.h

```diff
@@ -71,6 +71,8 @@ class SoftBody3DSW : public CollisionObject3DSW {
 	};
 
 	struct Face {
+		Vector3i nidx; // Nodal indices
```

Comment:
This extra member is not needed, you can use `face.n[0]`, `face.n[1]`, `face.n[2]` to access the nodes directly.